### PR TITLE
Update to Ruby 3.2.4

### PR DESCRIPTION
**Pull request by renchap**
It contains 3 security fixes, as well as a few bug fixes

https://www.ruby-lang.org/en/news/2024/04/23/ruby-3-2-4-released/

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ ARG TARGETPLATFORM=${TARGETPLATFORM}
 ARG BUILDPLATFORM=${BUILDPLATFORM}
 
 # Ruby image to use for base image, change with [--build-arg RUBY_VERSION="3.2.3"]
-ARG RUBY_VERSION="3.2.3"
+ARG RUBY_VERSION="3.2.4"
```
Comment (mjankowski) [inline Dockerfile:11]: There are a few other "3.2.3" strings in this file (in comments). Not critical to update, but maybe confusing for people to see diff values?

File: Dockerfile [hunk 2]
```diff
@@ -262,4 +262,4 @@ USER mastodon
 # Expose default Puma ports
 EXPOSE 3000
 # Set container tini as default entry point
-ENTRYPOINT ["/usr/bin/tini", "--"]
```
Comment (mjankowski) [inline Dockerfile:265]: Accidental newline removal here?

**Review by mjankowski — APPROVED** [review]
Left a few style comments, but looks fine overall ... although GH Actions does not have 3.2.4 available yet.

File: Dockerfile (hunk 2 above)
Comment (renchap) [reply Dockerfile:265]: My editor added a newline, which I think is the correct behaviour here.

**Comment by renchap** [comment]
Ruby 3.2.4 is now available in GH Actions, but not yet in the container images. Should be available in the next hour or so.

**Comment by renchap** [comment]
3.2.4 is now available in GH actions and docker base image, everything looks green!

**Review by ClearlyClaire — APPROVED** [review]

**Pull request merged**